### PR TITLE
doc

### PR DESCRIPTION
**Pull request by justinmk**
- [x] remove textprop references https://github.com/neovim/neovim/commit/aa680f6acb84b876584cbe5c958d50c692154c75#r34755682

### CODE REVIEW

**Comment by justinmk** [comment]
@blueyed thanks. JFI that doc is generated from the C sources :)

**Comment by zgpio** [comment]
There seems to be a redundant space here.
https://github.com/neovim/neovim/blob/6a21fe9fd16cfedc87b58064d09218e5c06ddd89/runtime/doc/api.txt#L715
https://github.com/neovim/neovim/blob/6a21fe9fd16cfedc87b58064d09218e5c06ddd89/runtime/doc/api.txt#L727

**Comment by justinmk** [comment]
> There seems to be a redundant space here.

It's not easy to fix. `scripts/gen_vimdoc.py` has to make some ambiguous decisions, the space is a tradeoff. And a pretty harmless one.

File: src/nvim/api/buffer.c [hunk 1]
```diff
@@ -181,7 +181,8 @@ Boolean nvim_buf_attach(uint64_t channel_id,
   return false;
 }
 
-/// Deactivates buffer-update events on the channel.
+/// Deactivates buffer-update events on the channel. Not usable from lua,
+/// for attached lua callbacks see |api-buffer-updates-lua-detach|.
```
Comment (justinmk) [inline src/nvim/api/buffer.c]: not in VimL either. I guess this could be in the Attributes section of the docs.

**Comment by zgpio** [comment]
Extra blank line
https://github.com/neovim/neovim/blob/1d59575450e0a09d710a7cad2b8e1b5c5e0242f5/runtime/doc/api.txt#L817-L819
https://github.com/neovim/neovim/blob/1d59575450e0a09d710a7cad2b8e1b5c5e0242f5/runtime/doc/api.txt#L1116-L1118
Affected the outline:
image

**Comment by zgpio** [comment]
It seems  `or 0 for current window` is missing.
https://github.com/neovim/neovim/blob/1d59575450e0a09d710a7cad2b8e1b5c5e0242f5/runtime/doc/api.txt#L1746-L1751

**Comment by blueyed** [comment]
Needs to be rebased (conflicts).
(wanted to push a minor fix for "wait()" (indent))

**Comment by justinmk** [comment]
Still more doc work needed, but merging this for now.

> Extra blank line ... Affected the outline:

@zgpio that will take time to fix (need to debug `gen_vimdoc.py`).

**Pull request merged**